src/client.ts

```diff
@@ -177,6 +177,7 @@ export default class Client {
 
   private registerAutoRestarts() {
     this.createRestartWatcher("Gemfile.lock");
+    this.createRestartWatcher(".rubocop.yml");
```

Comment:
Should we also restart if a config file changed (or was added/removed) in a sub-directory?